installer_fabfile.py

```diff
@@ -693,16 +680,70 @@ def install(pg_passwd='goldstone',  # pylint: disable=R0913
         install_repos()
         setup_postgres(pg_passwd)
         install_extra_rpms()
-        goldstone_init(
-            django_admin_user=django_admin_user,
-            django_admin_password=django_admin_password,
-            django_admin_email=django_admin_email,
-            gs_tenant=gs_tenant, gs_tenant_owner=gs_tenant_owner,
-            gs_tenant_admin=gs_tenant_admin,
-            gs_tenant_admin_password=gs_tenant_admin_password,
-            stack_tenant=stack_tenant, stack_user=stack_user,
-            stack_password=stack_password,
-            stack_auth_url=stack_auth_url)
+
+        syncmigrate()
+
+        django_admin_init(username=django_admin_user,
+                          password=django_admin_password,
+                          email=django_admin_email)
+
+        goldstone_init(gs_tenant=gs_tenant,
+                       gs_tenant_owner=gs_tenant_owner,
+                       gs_tenant_admin=gs_tenant_admin,
+                       gs_tenant_admin_password=gs_tenant_admin_password,
+                       stack_tenant=stack_tenant,
+                       stack_user=stack_user,
+                       stack_password=stack_password,
+                       stack_auth_url=stack_auth_url)
+
+    else:
+        print()
+        abort('This appears to be an unsupported platform. Exiting.')
+
+
+@task
+def install(django_admin_password,
+            pg_passwd='goldstone',
+            django_admin_user='admin',
+            django_admin_email='root@localhost'):
+    """Do a partial installation of Goldstone.
+
+    This requires no interaction with the user.
+
+    This does what full_install() does, except for creating the Goldstone
+    tenant and OpenStack cloud rows in the Tenant and Cloud tables. After this
+    runs, the user must log in to Goldstone as a Django admin, create a
```

Comment:
nits:
* refs to rows and tables are probably too much "sausage making." 
* as "the" Django admin?